packages/tasks/src/local-apps.ts

```diff
@@ -39,36 +45,52 @@ export type LocalApp = {
 			 * And if not (mostly llama.cpp), snippet to copy/paste in your terminal
 			 * Support the placeholder {{GGUF_FILE}} that will be replaced by the gguf file path or the list of available files.
 			 */
-			snippet: (model: ModelData) => string | string[];
+			snippet: (model: ModelData) => Snippet | Snippet[];
 	  }
 );
 
 function isGgufModel(model: ModelData) {
 	return model.tags.includes("gguf");
 }
 
-const snippetLlamacpp = (model: ModelData): string[] => {
+const snippetLlamacpp = (model: ModelData): Snippet[] => {
+	const command = (binary: string) =>
+		[
+			"# Load and run the model:",
+			`${binary} \\`,
+			'  --hf-repo "${model.id}" \\',
+			"  --hf-file {{GGUF_FILE}} \\",
+			'  -p "You are a helpful assistant" \\',
+			"  --conversation",
+		].join("\n");
 	return [
-		`# Option 1: use llama.cpp with brew
-brew install llama.cpp
-
-# Load and run the model
-llama \\
-	--hf-repo "${model.id}" \\
-	--hf-file {{GGUF_FILE}} \\
-	-p "I believe the meaning of life is" \\
-	-n 128`,
-		`# Option 2: build llama.cpp from source with curl support
-git clone https://github.com/ggerganov/llama.cpp.git 
-cd llama.cpp
-LLAMA_CURL=1 make
-
-# Load and run the model
-./main \\
-	--hf-repo "${model.id}" \\
-	-m {{GGUF_FILE}} \\
-	-p "I believe the meaning of life is" \\
-	-n 128`,
+		{
+			title: "Install from brew",
+			setup: "brew install llama.cpp",
```

Comment:
btw unrelated but wondering is llama.cpp is on winget? cc @mfuntowicz too

Reply:
I checked here: https://winget.run and didn't find any. To think of it, it'd be a pretty cool idea to add that.

Reply:
Yeah it would be nice to have, knowing that llama.cpp already have pre-built binary via CI. Unfortunately I'm not very familiar with windows stuff, so I'll create an issue on llama.cpp to see if someone can help.

Reply:
Created the issue: https://github.com/ggerganov/llama.cpp/issues/8188